averbis/core/_rest_client.py

```diff
@@ -2405,6 +2405,19 @@ def invalidate_api_token(self, user: str, password: str) -> None:
         )
         return None
 
+    def is_token_valid(self, api_token: str) -> bool:
+        """
```

Comment:
This method uses the clients pre-configured API token for the request, so it checks that one and not the one given as a parameter. In my opinion it would be better to remove the api_token paramter from this method to avoid confusion.